### PR TITLE
population structure lecture

### PR DESCRIPTION
**Pull request by LeandroRitter**

### CODE REVIEW

File: docs/slides/population_structure/index.qmd [hunk 1]
```diff
@@ -14,157 +19,786 @@ format:
...
+#| classes: my_code_size_abitsmaller
+cd /home/nikolay/Documents/Teaching/PopGen_2023/data/DavidReich
+head -8 AADR.snp
+```
 
-#### Nei
+```{bash}
+#| class-source: my_code_size_abitsmaller
+#| classes: my_code_size_abitsmaller
+cd /home/nikolay/Documents/Teaching/PopGen_2023/data/DavidReich
```
Comment (percyfal) [inline docs/slides/population_structure/index.qmd:512]: Ditto

File: docs/slides/population_structure/index.qmd [hunk 2]
```diff
@@ -14,157 +19,786 @@ format:
...
+Rscript jackKnife.R file=out.abbababa indNames=bams outfile=out_abbababa_results
+```
+
+and, finally, we will display the resulting table of D-statistics and
+plot the ABBA and BABA counts using custom R scripts:
+
+```{r}
+#| eval: true
+#| echo: false
+path<-"/home/nikolay/Documents/Teaching/PopGen_2023/data/"
```
Comment (percyfal) [inline docs/slides/population_structure/index.qmd:752]: Same as above; you would need to copy-paste the output, and also save an image of the boxplot.

**Comment by percyfal** [comment]
@LeandroRitter I added excerpts from the files to enable correct rendering but I had to change the file paths (e.g., for the Allen Ancient DNA Resource examples). Also, I ran pre-commit on the files, which basically does linting of code styles; this led to all R code blocks being modified. I don't think this has affected the final output in terms of, e.g., pushing images outside slide boundaries. Do a `git fetch -a` followed by `git merge origin/dev-nikolay` and rerender the pages to make sure everything is to your liking, then I can merge.

File: docs/slides/population_structure/index.qmd (hunk 2 above)
Comment (LeandroRitter) [reply docs/slides/population_structure/index.qmd]: @percyfal got it, will remove my local paths from the slides and insert the outputs as figures or text as you suggested above

**Review by percyfal — APPROVED** [review]
LGTM now that rendering works!

**Pull request merged**